### ISSUE TITLE
Custom F-Droid repo??

### ISSUE THREAD

**Issue by NatureHog** [blokadaorg/blokada#821]
Would it be possible to make a Custom F-Droid repo only for Blokada, so that the Blokada version could be updated around the same time as the release comes out?

Thanks in advance!

**Comment by peterroth** [comment]
No, it isn't possible. F-droid is an organisation and they review every update and app.

**Comment by NatureHog** [comment]
There is a possibility!

As an example: The popular Password Manager "Bitwarden" has a custom repo at https://mobileapp.bitwarden.com/fdroid/repo
(I haven't read much into how it works, I'm afraid...)

It is even possible to make a "fingerprint" to verify the repo.

Am 9. Jänner 2021 13:28:45 MEZ schrieb "Péter Róth" <notifications@github.com>:

**Comment by NatureHog** [comment]
Official docs about this here: https://f-droid.org/en/docs/Setup_an_F-Droid_App_Repo/

**Comment by peterroth** [comment]
Nice, thanks for sharing this information, I'll check it.

**Comment by NatureHog** [comment]
Any updates on this?

**Comment by Glitchy-Tozier** [comment]
Btw, NewPipe apparently also allows has this feature. (https://github.com/TeamNewPipe/NewPipe/issues/5421)

**Comment by mhmdanas** [comment]
I believe that Blokada doesn't need a custom repo. NewPipe is behind by many versions on the main F-Droid repo due to some issues related to reproducible builds (0.20.1 on the main F-Droid repo, 0.20.10 is the latest release), so having a custom repo is important. And even when the issue get fixed, NewPipe's updates are very time-sensitive, as changes in YouTube or other services might break NewPipe, and people don't really want to wait for several days until a fix is released. On the other hand, Blokada's updates aren't as time-sensitive as NewPipe. Since I believe custom repos should be avoided when there isn't a good reason for them, I think Blokada doesn't need one.

**Comment by kar** [comment]
Also, I have finally managed to make the automatic updates working for both v5 and v4 on fdroid, meaning the updates now do not need a review each time, and are automatically pulled. It still takes a few days before the update shows up on fdroid, but it's not weeks anymore.

Nevertheless I'll mark this task as a good first time task, if someone wants to volunteer to maintain the custom repo.

**Comment by KopfKrieg** [comment]
> Also, I have finally managed to make the automatic updates working for both v5 and v4 on fdroid

I can't find the current version of Blokada (both v4 and v5) in the official F-Droid repo any more. Has something changed regarding this?!

**Comment by peterroth** [comment]
v4 is available: https://f-droid.org/packages/org.blokada.alarm/
